modules/apps/foundation/user-associated-data/user-associated-data-api/src/main/java/com/liferay/user/associated/data/exporter/BaseUADEntityExporter.java

```diff
@@ -25,9 +25,6 @@
  */
 public abstract class BaseUADEntityExporter implements UADEntityExporter {
 
-	@Override
-	public abstract void export(UADEntity uadEntity) throws PortalException;
-
```

Comment:
We need to clean up other abstract classes too

Reply:
Oh yeah, good call. This was the only class I looked at during review so I spaced on making this change anywhere else.